code/ATMOSPHERICS/components/unary/vent_scrubber.dm

```diff
@@ -162,6 +162,8 @@
 			(scrub_N2O && environment[GAS_SLEEPING] > 0) ||\
 			(scrub_O2 && environment[GAS_OXYGEN] > 0) ||\
 			(scrub_N2 && environment[GAS_NITROGEN] > 0))
+			if (air_contents.return_pressure()>=50*ONE_ATMOSPHERE)
```

Comment:
Make `50*ONE_ATMOSPHERE` a define or something so you don't have to duplicate the value.

Reply:
done

Reply:
Why not a var?

Reply:
These are defines in all the other files, but I guess there's no reason it couldn't be a variable. I kind of want to do some pipecode refactoring later so maybe I'll just leave var-izing it for then.

Reply:
It definitely should be a var at some point but it doesn't have to be right now, so no problem